commands/system/toggle-airpods.template.swift

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/swift
+
+// Required parameters:
+// @raycast.schemaVersion 1
+// @raycast.title Toggle AirPods
+// @raycast.mode silent
+// @raycast.packageName Audio
+//
+// Optional parameters:
+// @raycast.icon images/airpod.png
+//
+// Documentation:
+// @raycast.description Toggle AirPods bluetooth device
+// @raycast.author Nichlas W. Andersen
+// @raycast.authorURL https://github.com/itsnwa
+
+import IOBluetooth
+
+func toggleAirPods() {
+    guard let bluetoothDevice = IOBluetoothDevice(addressString: "Your AirPods MAC address") else {
```

Comment:
Let's extract the address to a constant and add a comment how to get it.

```suggestion

// Get your device's MAC address by option (⌥) + clicking the bluetooth icon in the menu bar
let deviceAddress = ""

func toggleAirPods() {
    guard let bluetoothDevice = IOBluetoothDevice(addressString: deviceAddress) else {
```